lib/phlex/sgml.rb

```diff
@@ -105,6 +105,11 @@ def __final_call__(buffer = +"", context: Phlex::Context.new, view_context: nil,
 			@_context = context
 			@_view_context = view_context
 			@_parent = parent
+			if @_rendered
+			  warn "⚠️ [WARNING] You are rendering a component #{self.class.name} twice. This is not supported in Phlex 2.0."
+			  @_rendered = true
```

Comment:
I think this assignment was meant to be outside the `if` condition.

Reply:
oops, right 😅